internal/provider/resource_http_health.go

```diff
@@ -121,6 +123,15 @@ func (*HttpHealthResource) Schema(ctx context.Context, req resource.SchemaReques
 				Optional:            true,
 				MarkdownDescription: "If false, the resource will fail to create if the check does not pass. If true, the resource will be created anyway. Defaults to false.",
 			},
+			"ca_bundle": schema.StringAttribute{
+				Optional:            true,
+				MarkdownDescription: "The CA bundle to use when connecting to the target host.",
+			},
+			"insecure_tls": schema.BoolAttribute{
+				Optional:      true,
+				Computed:      true,
+				PlanModifiers: []planmodifier.Bool{modifiers.DefaultBool(false)},
```

Comment:
If the default is false you don't actually need a planmodifier, since the null value maps to false